scripts/build-shotcut.sh

```diff
@@ -20,10 +20,13 @@ SOURCE_DIR="$INSTALL_DIR/src"
 ACTION_GET_COMPILE_INSTALL=1
 ACTION_GET_ONLY=0
 ACTION_COMPILE_INSTALL=1
+TAR_APP_PACKAGE=1
```

Comment:
Hmm, right now, TAR_APP_PACKAGE and KEEP_BUILD_FILES would be Linux-only options. This script really has startup-scripts, archiving, and cleanup co-mingled. That could be split up, but that is also a lot to change and test and a separate issue. Also, skipping the "deploy" step is of dubious value on Windows and OS X since you cannot really use them without it. Also, I have felt the current ACTION_ options are rather awkward, and perhaps all this should be:
ACTION_CLEAN_SOURCES
ACTION_GET
ACTION_COMPILE
ACTION_INSTALL
ACTION_ARCHIVE
ACTION_CLEAN_INSTALL

Now, with all that said, I think that set of changes is well outside the scope of metellius' itch. :o) So, I am inclined to accept this as-is hoping to change it soon. IOW, while I kind of agree with Brian about the name, I do not think that is worth changing the commit especially when it is due to be changed with a general cleanup anyways.